config/initializers/lograge.rb

```diff
@@ -0,0 +1,20 @@
+Rails.application.configure do
+  unless ENV["DISABLE_LOGRAGE"].present?
+    config.lograge.enabled = true
+  end
+
+  config.lograge.custom_options = lambda do |event|
+    exceptions = %w(controller action format id)
+    {
+      params: event.payload[:params].except(*exceptions),
+      time: event.time.utc().iso8601(3),
```

Comment:
Cool. Do you think this time format is what we'll standardize on for centralized logging?

Reply:
I don't really have a strong argument for this other than it's what I prefer.